src/index.js

```diff
@@ -270,8 +270,12 @@ export default class Undo {
         this.blocks.delete();
         this.caret.setToBlock(index, "end");
       } else if (blockCount > state.length) {
+        let newState = state
+        if(state.length==0) {
+          newState =  [{ type: "paragraph", data: {} }]
+        }
```

Comment:
@khanakia IMO we could use a ternary operator! ⬇️ 
```suggestion
        const newState = state.length === 0 ? [{ type: 'paragraph', data: {} }] : state;

```